TrackletAlgorithm/MatchEngineUnit.h

```diff
@@ -448,6 +436,10 @@ inline void advance() {
  ProjectionRouterBuffer<VMProjType, AllProjectionType> projbuffer_;
  ap_uint<kNBits_MemAddr> projseq_;
  bool isPSseed_;
+ ap_uint<1> usefirstMinus;
```

Comment:
Use trailing _ for member data.

Reply:
Actually - as far as I can tell these are only used in one method, init(), so they should not be class variables, but local variables in the init method.

Reply:
Good catch, I've moved them.